code/game/objects/structures/ai_core.dm

```diff
@@ -12,6 +12,7 @@
 	var/datum/ai_laws/laws
 	var/obj/item/circuitboard/aicore/circuit
 	var/obj/item/mmi/core_mmi
+	var/mob/living/silicon/ai/remote_ai = null // only used in cases of AIs piloting mechs or shunted malf AIs, possible later use cases
```

Comment:
Auto doc

Reply:
Oh I see, I'll have to look at auto doc more